backend/btrixcloud/migrations/migration_0016_operator_scheduled_jobs_v2.py

```diff
@@ -0,0 +1,58 @@
+"""
+Migration 0016 - Updating scheduled cron jobs after Operator changes v2
+"""
+from btrixcloud.models import CrawlConfig, UpdateCrawlConfig
+from btrixcloud.crawlmanager import CrawlManager
+from btrixcloud.migrations import BaseMigration
+
+
+MIGRATION_VERSION = "0016"
+
+
+class Migration(BaseMigration):
+    """Migration class."""
+
+    def __init__(self, mdb, migration_version=MIGRATION_VERSION):
+        super().__init__(mdb, migration_version)
+
+    async def migrate_up(self):
+        """Perform migration up.
+
+        Find existing workflows with schedule and create new crawl_cron_jobs
+        from template, back in crawlers workspace and using noop image
+        """
+        # pylint: disable=too-many-locals, duplicate-code
+        crawl_configs = self.mdb["crawl_configs"]
+        crawl_manager = CrawlManager()
+
+        # Update configmap for crawl configs that have non-zero timeout or scale > 1
+        match_query = {"schedule": {"$nin": ["", None]}}
+        configs_to_update = [res async for res in crawl_configs.find(match_query)]
+        for config_dict in configs_to_update:
+            config = CrawlConfig.from_dict(config_dict)
+            print(
+                f"Updating CronJob for Crawl Config {config.id}: schedule: {config.schedule}"
+            )
+            try:
+                await crawl_manager.update_crawl_config(
+                    config,
+                    UpdateCrawlConfig(
+                        schedule=config.schedule,
+                    ),
+                )
+            # pylint: disable=broad-except
+            except Exception as exc:
+                print(
+                    "Skip crawl config migration due to error, likely missing config",
+                    exc,
+                )
+
+        # Delete existing scheduled jobs from default namespace
+        print("Deleting cronjobs from default namespace")
+        await crawl_manager.batch_api.delete_collection_namespaced_cron_job(
+            namespace="default", label_selector="btrix.crawlconfig"
```

Comment:
Hm, yeah good point! Removed the CRON_NAMESPACE setting as its now the same as crawl namespace, but it was the release namespace, can add that back in just in case, for anyone not running in `default` namespace

Reply:
Updated to pass in DEFAULT_NAMESPACE for the namespace used for release, and use that for migration. Will work if someone has been running in a namespace other than `default`.